content/docs/authorization/auth-code-flow/index.md

```diff
@@ -1,493 +1,90 @@
 ---
-layout: layout-sidebar
 name: Authorization Code Flow
 description: The Authorization Code Flow is appropriate for applications that the user logs into once. Since this flow involves an exchange of your **Application secret* for an access token, it is suitable for applications than run from secure locations such as server-side web application or back-end service.
+layout: layout-tutorial
 order: 100
 published: true
 showInNav: true
-back_to_top: true
 title: Authorization Code Flow
+sidebarLayoutSecondaryColumn: col-md-2 sidebar-tutorial
+sidebarLayoutPrimaryColumn: col-md-8
+bootstrap_container: bg-tutorial
+markdown: false
```

Comment:
Just a thought, but I wonder if instead of converting this page HTML if we could add the structure to the markdown converter to make this easier to build and keep consistent. I think we already do this elsewhere w/ our custom `blackbaud-marked` that we're using in stache. Not sure if this is hook-able into stache or not... @Blackbaud-BobbyEarl might know...

http://beneb.info/2015/02/more-powerful-markdown-with-custom-renderers/